scripts/start.sh

```diff
@@ -5,9 +5,24 @@ set -e
 CUR_DIR=$(cd "$(dirname "$0")" && pwd)
 BASE_DIR=$(dirname "$CUR_DIR")
 
+min() {
+    echo $(( $1 < $2 ? $1 : $2 ))
+}
+
+workers=$(min 30 ${NB_WORKERS_ANTARES:-17}) # default to 17 and max is 30
```

Comment:
Did not really follow the subject but why did we choose these values ?

Reply:
- The default `17` is when you don't provide `NB_WORKERS_ANTARES` in your docker-compose env vars
- In order to have a reserved range of ports for `antarest`, we need to have a maximum of reserved ports on your server. It follows that you can't have more workers than ports.

Reply:
I would prefer to not hard code here the 17 because it's related to our hardware.

We could use `nproc` to keep the same logic as before (2*nproc + 1), or just throw an error if the variable does not exist.

Reply:
@sylvlecl  
I don't know if the value `(2*nproc + 1)` would be the same on deployment server and `CI/CD` pipeline server. 
So it is better to hard code it.